packages/react-native/Libraries/AppDelegate/React-RCTAppDelegate.podspec

```diff
@@ -25,7 +25,7 @@ use_hermes = ENV['USE_HERMES'] == nil || ENV['USE_HERMES'] == '1'
 
 new_arch_enabled_flag = (is_new_arch_enabled ? " -DRCT_NEW_ARCH_ENABLED" : "")
```

Comment:
```suggestion
new_arch_enabled_flag = (is_new_arch_enabled ? " -DRCT_NEW_ARCH_ENABLED=1" : "")
```

Reply:
Applied with minor tweak to use `\=1`.

Reply:
i was checking existing code and found this
https://github.com/facebook/react-native/blob/3e6b4fa23088e92fb41608fa7dbe3cf410626fb3/packages/react-native/React/React-RCTFabric.podspec#L23

why do we need to escape the `=`?

Reply:
You're right. I could have sworn I saw that escape syntax elsewhere but nope. Fixed and rebased.